File: modeshape-repo/modeshape-jcr-rest-boot/src/main/java/com/bpwizard/wcm/repo/rest/jcr/model/FormTab.java
package com.bpwizard.wcm.repo.rest.jcr.model;

public class FormTab {
	private String tabName;
	private BaseFormGroup[] formGroups;
	public String getTabName() {
		return tabName;
	}
	public void setTabName(String tabName) {
		this.tabName = tabName;
	}
	public BaseFormGroup[] getFormGroups() {
		return formGroups;
	}
	public void setFormGroups(BaseFormGroup[] formGroups) {
		this.formGroups = formGroups;
	}
}

File: ui/angular/projects/bpw-wcm-service/src/lib/model/FormTab.ts
import { BaseFormGroup } from "./BaseFormGroup";
import { VisbleCondition } from "./VisbleCondition";
export interface FormTab {
  tabName: string;
  tabTitle?: string;
  order?: number;
  formGroups: BaseFormGroup[];
  consition?: VisbleCondition;
}
